lib/Constants.js

```diff
@@ -183,6 +183,11 @@ HttpStatus.init({
   },
 });
 
+const KeyCode = {
```

Comment:
can we document the purpose of this?  in general `Constants` has very little documentation, despite being the most widely used / referred-to module.